maths/radians.py

```diff
@@ -0,0 +1,24 @@
+from math import pi
+'''
+
+Coverts the given angle from degrees to radians
+https://en.wikipedia.org/wiki/Radian
+'''
+def radians(degree):
+
+    """
+    >>> radians(180)
+    3.141592653589793
+    >>> radians(92)
+    1.6057029118347832
+    >>> radians(274)
+    4.782202150464463
```

Comment:
When replicating a Python Standard Library function, please try to do side-by-side tests...

Add tests:
```
>>> from math import radians as math_radians
>>> all(radians(i) == math_radians(i) for i in range(-2, 361))
True
```
Test should always try negative numbers and zeros.  Testing for floating point numbers is also good idea.

Reply:
Test was added